crates/turbopack-node/src/render/node_api_source.rs

```diff
@@ -64,6 +66,7 @@ pub struct NodeApiContentSource {
     route_match: RouteMatcherVc,
     entry: NodeEntryVc,
     render_data: JsonValueVc,
+    debug: bool,
```

Comment:
Could this be integrated to tracing instead?

Reply:
yeah it's for starting node.js in debug mode